packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -224,6 +242,18 @@ const classComponentUpdater = {
 
     enqueueUpdate(fiber, update);
     scheduleUpdateOnFiber(fiber, expirationTime);
+
+    if (__DEV__) {
+      if (enableDebugTracing) {
+        const priorityLevel = inferPriorityFromExpirationTime(
```

Comment:
Since this is DEV-only, you can read the `.priorityLevel` off the update object, without having to infer based on expiration time.

Reply:
Ooh, good tip.